modules/sdk-coin-zketh/test/unit/zkethToken.ts

```diff
@@ -0,0 +1,32 @@
+import 'should';
+
+import { TestBitGo, TestBitGoAPI } from '@bitgo/sdk-test';
+import { ZkethToken } from '../../src';
+import { BitGoAPI } from '@bitgo/sdk-api';
+
+describe('Zketh Token:', function () {
+  let bitgo: TestBitGoAPI;
+  let zkethTokenCoin;
+  const tokenName = 'tzketh:usdt';
+
+  before(function () {
+    bitgo = TestBitGo.decorate(BitGoAPI, { env: 'test' });
+    ZkethToken.createTokenConstructors().forEach(({ name, coinConstructor }) => {
+      bitgo.safeRegister(name, coinConstructor);
+    });
+    bitgo.initializeTestVars();
+    zkethTokenCoin = bitgo.coin(tokenName);
+  });
+
+  it('should return constants', function () {
+    zkethTokenCoin.getChain().should.equal('tzketh:usdt');
+    zkethTokenCoin.getBaseChain().should.equal('tzketh');
+    zkethTokenCoin.getFullName().should.equal('Zketh Token');
+    zkethTokenCoin.getBaseFactor().should.equal(1e18);
+    zkethTokenCoin.type.should.equal(tokenName);
+    zkethTokenCoin.name.should.equal('zkSync Test USDT');
```

Comment:
Need to check if we are able to get zksync usdt or not from faucet, otherwise for testing we will have to add some other token

Reply:
I see you have updated it to link